polaris-react/src/components/Box/Box.stories.tsx

```diff
@@ -33,56 +33,40 @@ export function WithBorders() {
       <Box
         background="bg"
         padding="400"
-        borderWidth="2"
+        borderWidth="050"
         borderColor="border-info"
       >
         <Text as="p">2px solid blue</Text>
       </Box>
       <Box
         background="bg"
         padding="400"
-        borderWidth="3"
-        borderColor="border-success"
-      >
-        <Text as="p">3px solid green</Text>
-      </Box>
```

Comment:
Removed the `borderWidth="3"` example since `--p-border-width-3` becomes `--p-border-width-100` and that's redundant with the `borderWidth="4"` example which maps to `--p-border-width-100` as well.